src/lib/utils/formatLocaleNumber.ts

```diff
@@ -0,0 +1,23 @@
+import { Currency, CurrencyAmount, Price } from '@uniswap/sdk-core'
+import { DEFAULT_LOCALE, SUPPORTED_LOCALES } from 'constants/locales'
+
+interface FormatLocaleNumberArgs {
+  number: CurrencyAmount<Currency> | Price<Currency, Currency> | number
+  locale: string | null | undefined
+  options?: Intl.NumberFormatOptions
+  sigFigs?: number
+}
+
+export default function formatLocaleNumber({ number, locale, sigFigs, options = {} }: FormatLocaleNumberArgs): string {
+  if (locale && !SUPPORTED_LOCALES.includes(locale)) {
+    locale = DEFAULT_LOCALE
```

Comment:
You use `locale` as `[locale, DEFAULT_LOCALE]` below, so just transform it here.

```
if (locale && SUPPORTED_LOCALES.includes(locale)) {
  locale = [locale, DEFAULT_LOCALE]
} else {
  locale = DEFAULT_LOCALE
}
```